hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestSplitTransaction.java

```diff
@@ -277,7 +323,7 @@ public void testWholesomeSplit() throws IOException {
         assertTrue(count > 0 && count != rowcount);
         daughtersRowCount += count;
       } finally {
-        HRegion.closeHRegion((HRegion)openRegion);
```

Comment:
`HRegion.closeHRegion` method will close WAL, then second daughter region will get exception. That's why i replace it with `((HRegion) openRegion).close()`

Reply:
Makes sense! 👍